resource.html

```diff
@@ -0,0 +1,1297 @@
+                                <img src="./assets/post-article.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="logo">
+                              </div>
+                            </div>
+
+                            <div class="teaser aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                              <div class="cmp-teaser">
+                                <div class="cmp-teaser__image">
+                                  <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_273228732/container/container_254357149/teaser.coreimg{.width}.png/1669068677351/latest-post1.png" data-asset="/content/dam/LeaderImpact/site-./assets/latest-post1.png" data-asset-id="0b6ddea4-6648-4057-abf4-e1c3c7c5e4c8" id="image-cd419a3839" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                                    <a class="cmp-image__link" href="/content/leaderimpact/lang/en2/resources/leading-forward-series.html" data-cmp-hook-image="link">
+                                      <img src="./assets/latest-post1.png" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="">
+                                    </a>
+                                  </div>
+                                </div>
+
+                                <div class="cmp-teaser__content">
+                                  <h2 class="cmp-teaser__title">
+                                    <a class="cmp-teaser__title-link" href="/content/leaderimpact/lang/en2/resources/leading-forward-series.html">Living the Fullfilled Life of a Leader</a>
+                                  </h2>
+                                  <div class="cmp-teaser__description"><p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna.</p>
+                                  </div>
+                                </div>
+                              </div>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+
+            </div>
+
+            <div class="container responsivegrid article-slider aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                <div id="container-b0c5fbd9e2" class="cmp-container">
+                  <div id="slider-container" class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
```

Comment:
You can't use this id more than once.

Reply:
Need to figure this out as the slider are being repeated in resource page.